playbooks/roles/os_temps/tasks/install_oc_bin.yml

```diff
@@ -1,34 +1,48 @@
 ---
+# Check if user already has the right version of `oc` installed in path
+- name: "Check if `oc` with version: {{ oc_version }} is in path"
+  shell: "type oc && oc version | grep -q {{ ('^oc ' ~ oc_version) | quote }}"
```

Comment:
While this command is checking the oc bin that's in path, the rest of the playbook uses the binary defined in  `oc_bin` variable which is not the same unless specifically configured.

Reply:
Sorry, missed the set_fact below